docs/adr/0001-provide-mechanism-to-show-hide-sections-of-accessibility-statement.md

```diff
@@ -0,0 +1,108 @@
+# Provide mechanism for showing/hiding/updating parts of the accessibility statement
+
+* Status: proposed
+* Date: 2022-02-08
+
+Technical Story: [PLATUI-1552](https://jira.tools.tax.service.gov.uk/browse/PLATUI-1552)
+
+## Context and Problem Statement
+
+Currently, some content within the existing accessibility statement template is not suitable for VOA, C-HGV, and GVMS services.
+The current implementation of the accessibility statement does not allow for hiding/showing or changing certain sections of the accessibility statement
+that some teams may require.
+
+## Decision Drivers <!-- optional -->
+
```

Comment:
Some more decision drivers we could add:

- as much as possible we want to make it impossible for a service create a statement that doesn't meet legal requirements
- we want to avoid making use of the service more complicated for services that don't shouldn't be varying their accessibility statements

Reply:
One more decision driver:
• As service maintainers, we need to find the correct balance between flexibility and code readability, otherwise it will be increasingly difficult to maintain the service in a timely and reliable way